ee/ucp/authorization/grant-permissions.md

```diff
@@ -22,51 +22,69 @@ A common workflow for creating grants has four steps:
 - Group cluster **resources** into Swarm collections or Kubernetes namespaces.
 - Create **grants** by combining subject + role + resource set.
 
-## Kubernetes grants
+## Creating grants
+ To create a grant:
+ 
+ 1. Log in to the UCP web UI.
+ 2. Click **Access Control**.
+ 3. Click **Grants**.
+ 4. In the Grants window, select **Kubernetes** or **Swarm**.
+
+### Kubernetes grants
 
 With Kubernetes orchestration, a grant is made up of *subject*, *role*, and
 *namespace*.
 
+> Note
```

Comment:
should we update these on master to be like

```suggestion
> **Note**:
```

Reply:
@traci-morrison - Could you fix this on the master? Thanks.